### PR TITLE
Add eslint + import formatter

### PR DESCRIPTION
**Pull request by dsiddharth**

### CODE REVIEW

File: commands/cloud/destroy.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { CommandModule } from "yargs";
+import prompts from "prompts";
+import { makeCloudApiRequest } from "../../utils";
+
+const cmd: CommandModule = {
+  command: "destroy",
+  describe: "Destroy a Hathora Cloud application",
+  builder: {
+    yes: { type: "boolean", describe: "Accept all confirmations" },
```
Comment (hpx7) [inline commands/cloud/destroy.ts]: should we explicitly add a `default: false` here?

File: utils.ts [hunk 2]
```diff
@@ -9,6 +9,28 @@ import shelljs from "shelljs";
...
+      },
+      responseType: "stream",
+    });
+
+    response.data.pipe(process.stdout);
+  } catch (err) {
+    if (axios.isAxiosError(err)) {
+      (err.response?.data as Stream).on("data", (data) => console.error(data.toString()));
+    }
+  }
```
Comment (hpx7) [inline utils.ts]: should probably log the err if it's not an axois error (also in deploy)

File: commands/cloud/destroy.ts (hunk 1 above)
Comment (dsiddharth) [reply commands/cloud/destroy.ts]: done

File: utils.ts (hunk 2 above)
Comment (dsiddharth) [reply utils.ts]: done

**Pull request closed without merging**